pkgs/by-name/qb/qbittorrent/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  apple-sdk_11,
+  boost,
+  cmake,
+  dbus,
+  fetchFromGitHub,
+  guiSupport ? true,
+  hicolor-icon-theme,
+  lib,
+  libtorrent-rasterbar,
+  nix-update-script,
+  openssl,
+  pkg-config,
+  python3,
+  qt6,
+  stdenv,
+  trackerSearch ? true,
+  webuiSupport ? true,
+  wrapGAppsHook3,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "qbittorrent" + lib.optionalString (!guiSupport) "-nox";
+  version = "5.0.2";
+
+  src = fetchFromGitHub {
+    owner = "qbittorrent";
+    repo = "qBittorrent";
+    rev = "release-${finalAttrs.version}";
+    hash = "sha256-JIURzAkVNYjHAx8yY0WaKNK4N/z9ndqjk0EXkJH9AzY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    wrapGAppsHook3
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs =
+    [
+      boost
+      libtorrent-rasterbar
+      openssl
+      qt6.qtbase
+      qt6.qtsvg
+      qt6.qttools
+      zlib
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_11 ]
+    ++ lib.optionals guiSupport [ dbus ]
+    ++ lib.optionals (guiSupport && stdenv.hostPlatform.isLinux) [ qt6.qtwayland ]
+    ++ lib.optionals trackerSearch [ python3 ];
+
+  cmakeFlags =
+    [ "-DVERBOSE_CONFIGURE=ON" ]
+    ++ lib.optionals (!guiSupport) [
+      "-DGUI=OFF"
+      "-DSYSTEMD=ON"
+      "-DSYSTEMD_SERVICES_INSTALL_DIR=${placeholder "out"}/lib/systemd/system"
+    ]
+    ++ lib.optionals (!webuiSupport) [ "-DWEBUI=OFF" ];
+
+  qtWrapperArgs = lib.optionals trackerSearch [ "--prefix PATH : ${lib.makeBinPath [ python3 ]}" ];
+
+  dontWrapGApps = true;
+
+  postInstall = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    APP_NAME=qbittorrent${lib.optionalString (!guiSupport) "-nox"}
+    mkdir -p $out/{Applications,bin}
+    mv $out/$APP_NAME.app $out/Applications
+    makeWrapper $out/{Applications/$APP_NAME.app/Contents/MacOS,bin}/$APP_NAME
+  '';
+
+  preFixup = ''
+    qtWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
```suggestion
  passthru.updateScript = nix-update-script { extraArgs = [ "--version-regex=release-(.*)" ]; };
```

Reply:
Run `nix-shell maintainers/scripts/update.nix --argstr package qbittorrent` to test.